src/sagas/index.js

```diff
@@ -0,0 +1,51 @@
+import { all, call, takeEvery, put } from 'redux-saga/effects';
```

Comment:
I would separate these sagas in individual files to keep it organized, imagine if this project grows, it wouldn't be maintainable to keep all sagas in one file.